api_v3/services/MediaService.php

```diff
@@ -702,10 +702,11 @@ function getAction($entryId, $version = -1)
      * @action getMrss
      * @param string $entryId Entry id
      * @param KalturaExtendingItemMrssParameterArray $extendingItemsArray
+     * @param string $features
      * @return string
      * @throws KalturaErrors::ENTRY_ID_NOT_FOUND
      */
-    function getMrssAction($entryId, KalturaExtendingItemMrssParameterArray $extendingItemsArray = null)
+    function getMrssAction($entryId, KalturaExtendingItemMrssParameterArray $extendingItemsArray = null, $features = null)
```

Comment:
I think the features are internal and used only when the MRSS requested internally.
In what use case do we need to expose it in the API? If it's used only by the admin-console XSLT tester, please add the feature in the action implementation when admin-console master partner is used.